### DISCUSSION TITLE
How to warm-start DeepAR from a previously-trained DeepAR model?

### DISCUSSION THREAD

**Discussion by la-cruche** [awslabs/gluonts#2346] [Q&A] [answered]
How to warm-start DeepAR from a previously-trained DeepAR model?

**Comment by lostella** [comment, marked as answer]
Incremental training was recently added: https://github.com/awslabs/gluonts/pull/2249

An example in the PR shows how to do that in Python; if you're running on SageMaker with a container based on the [`gluonts.shell`](https://github.com/awslabs/gluonts/tree/dev/src/gluonts/shell) package, then you can provide an additional `model` channel pointing to a previous job's output.

Note: this is only available on the `dev` branch as of today, but will be released with `v0.11.0` shortly.
